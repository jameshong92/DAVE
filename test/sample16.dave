// test field defintion
int main() {
	str[] s = ["a", "b", "c", "d", "e"];
	fld test = new fld(s, "alphabet");
	print(test);
  test = new fld(["a", "b", "c", "d", "e"], "alphabet");
  print(test);
  print(test.length);
}