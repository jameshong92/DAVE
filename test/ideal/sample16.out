alphabet	
a	
b	
c	
d	
e	

alphabet	
a	
b	
c	
d	
e	

5
